website/intro/use-cases.html.markdown

```diff
@@ -3,28 +3,21 @@ layout: "intro"
 page_title: "Use Cases"
 sidebar_current: "use-cases"
 description: |-
-  Before understanding use cases, it's useful to know what Terraform is. This page lists some concrete use cases for Terraform, but the possible use cases are much broader than what we cover. Due to its extensible nature, providers and provisioners can be added to further extend Terraform's ability to manipulate resources.
+  Learn common use cases for Terraform including Heroku apps, self-service clusters, and multi-cloud deployment. 
 ---
 
 # Use Cases
 
-Before understanding use cases, it's useful to know [what Terraform is](/intro/index.html).
-This page lists some concrete use cases for Terraform, but the possible use cases are
-much broader than what we cover. Due to its extensible nature, providers and provisioners
+This page lists a subset of use cases for [Terraform](/intro/index.html). Due to its extensible nature, providers and provisioners
```

Comment:
I think the second sentence is talking about building your own custom providers, but it kind of makes it sound like you only need a provider or provisioner if you want to do more with Terraform than the binary does alone.

I think we could either delete this line since this is beginner content and writing plugins is advanced, or just clarify that you can write your own Terraform plugins if you have a use case that the currently available ones don't cover.

Reply:
Done - I've removed the sentence. I agree - we don't need to go into that here, since this is intro stuff.